extension/intents/pocket/contentScript.js

```diff
@@ -0,0 +1,141 @@
+javascript: (function() {
```

Comment:
I think this technically works with `javascript:` at the beginning because it looks like a label... but it should probably be removed.

Also we should include a comment about where the code came from (just giving the URL where it was found)